[MTM-55504] add entry about custom web sdk registry (#1414)
SoftwareAG/c8y-docs: content/web/libraries.md

section: 
  - app_development
---

{{< c8y-admon-info >}}

> Because of an ongoing issue with npm, we are since 01.08.2023 unable to publish new releases of the Web SDK package `@c8y/ngx-components` there.

Reviewer: ```suggestion
Since August, 1st 2023, we are unable to publish new releases of the Web SDK package `@c8y/ngx-components` due to an ongoing issue with npm.
```

Until the issue with npm is resolved the node packages can be loaded from a another registry.

> To configure this additional registry for the packages prefixed with `@c8y`, a file with the name `.npmrc` and the following content should be present in your project directory:

Reviewer: ```suggestion
To configure this additional registry for the packages prefixed with `@c8y`, a file with the name `.npmrc` and the following content must be present in your project directory:
```

```
registry=https://registry.npmjs.org/
@c8y:registry=https://download.cumulocity.com/npm/
```

> The scaffolding process via `c8ycli` must already include this additional file depending on the version that you use.

Author: ```suggestion
The scaffolding process via `c8ycli` will already include this additional file depending on the version that you use.
```
